### ISSUE TITLE
Option to order tags

### ISSUE THREAD

**Issue by m8than** [louislam/uptime-kuma#2408]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

UI Feature

### 🔖 Feature description

I'd like to see an option to order tags wherever they're displayed. I believe it'd make the dashboard more organised and easier to read.

At the moment as you can see they're ordered in whichever order they are added
![image](https://user-images.githubusercontent.com/23742732/207584064-855a1f31-0511-4b98-b678-9f7a8efe0cd0.png)

### ✔️ Solution

In my use case, maybe this could change the default behaviour of displaying tags (or literally just a checkbox in settings).

Just so tags line up and make it easier to read.

**Comment by Blboun3** [comment]
I agree that this would be very useful and I'd like to see it as well!
In the mean time, have you found any solution ?
